PopcornEditor/src/butter.js

```diff
@@ -922,67 +922,127 @@ window.Butter = {
           finishedCallback( project );
         }
 
+        function initialMediaSources( qs ){
```

Comment:
Filed https://github.com/mozilla/popcorn-editor/issues/51 to capture this notion. Might need separate event pairs for config & initial-project-data, or might do those together? (Need to be able to put config data in early to do things like enable/disable plugins, I think)